app/src/main/java/ch/blinkenlights/android/vanilla/PlaybackActivity.java

```diff
@@ -793,7 +797,11 @@ public void onClick(DialogInterface dialog, int which) {
 							request.putExtra(PluginUtils.EXTRA_PARAM_SONG_TITLE, resolved.title);
 							request.putExtra(PluginUtils.EXTRA_PARAM_SONG_ARTIST, resolved.artist);
 							request.putExtra(PluginUtils.EXTRA_PARAM_SONG_ALBUM, resolved.album);
-							startService(request);
+							if (request.resolveActivity(getPackageManager()) != null) {
+								startActivity(request);
+							} else {
+								Log.e("PluginSystem", "Couldn't start plugin activity for " + request);
```

Comment:
Should we make this a toast message?

Reply:
It'd be a duplicate. The toast is shown in `checkPlugins`, it's visible when you open PlaybackActivity or create a context menu for an item in LibraryActivity.